### PR TITLE
add treatment follow up nhdd code mappings

### PR DESCRIPTION
**Pull request by freddieptf**
Part of https://github.com/moh-kenya/echis-2.0-integration/issues/110
Add nhdd code mappings for treatment follow up and under 5 assessment form

### CODE REVIEW

**Review by kitsao — CHANGES_REQUESTED** [review]
Good progress @freddieptf. It will be there in no time. I have shared a few comments inline.

File: src/utils/nhdd_mappings.js [hunk 1]
```diff
@@ -100,6 +105,329 @@ const codes = {
...
+    // treatment followup
+    better: {
+        system: NHDD_GENERIC_PATH,
+        code: "23831",
+        display: "Better",
+    },
+    worse: {
+        system: NHDD_GENERIC_PATH,
+        code: "28847",
+        display: "Deterioration",
```
[the remark this answers is not in the document]
Comment (freddieptf) [reply src/utils/nhdd_mappings.js:117]: They are all strings regardless

File: src/utils/nhdd_mappings.js [hunk 2]
```diff
@@ -145,7 +501,42 @@ const developmentalMilestones = {
...
+    fatigue: codes.fatigue,
+    headache: codes.headache,
+    hives: codes.hives,
+    insomnia: codes.insomnia,
+    irregular_heartbeat: codes.irregularHeartbeat,
+    nausea: codes.nausea,
+    upset_stomach: codes.upsetStomach,
+    rash: codes.rash,
+    vomiting: codes.vomiting
+};
```
[the remark this answers is not in the document]
Comment (freddieptf) [reply src/utils/nhdd_mappings.js:527]: Yes but we'd need to review the current forms and make sure the diagnosis labels we include in the screening object are the same globally

[the remark this answers is not in the document]
Comment (freddieptf) [reply src/utils/nhdd_mappings.js:527]: > Can use a function for this? It shouldn't be complicated, right?

What did you have in mind?

File: src/utils/nhdd_mappings.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kitsao) [reply src/utils/nhdd_mappings.js:117]: Yeah, but remember we set the rules, we stick to them.

**Review by kitsao — APPROVED** [review]
As discussed @freddieptf, lets refactor later, complete with tests. UAT already passed.

**Pull request merged**